python/lsst/pipe/base/pipelineBuilder.py

```diff
@@ -55,123 +50,83 @@
 
 
 class PipelineBuilder(object):
-    """
-    PipelineBuilder class is responsible for building task pipeline from
-    a set of command line arguments.
+    """PipelineBuilder class is responsible for building task pipeline.
+
+    The class provides a set of methods to manipulate pipeline by adding,
+    deleting, re-ordering tasks in pipeline and chaning their labels or
+    configuration.
 
     Parameters
     ----------
     taskFactory : `TaskFactory`
         Factory object used to load/instantiate PipelineTasks
+    pipeline : `Pipeline`, optional
+        Initial pipeline to be modified, if `None` then new empty pipeline
+        will be created.
     """
+    def __init__(self, taskFactory, pipeline=None):
+        if pipeline is None:
+            pipeline = Pipeline()
+        self._taskFactory = taskFactory
+        self._pipeline = pipeline
 
-    def __init__(self, taskFactory):
-        self.taskFactory = taskFactory
+    def pipeline(self, order_pipeline=False):
```

Comment:
Keyword arguments in this corner of the codebase should be camelCase.  Given that "pipeline" is already the method name, though, I'd just called this argument `ordered`.

Reply:
Yes, I believe so.